src/ConsoleApp/consoleAppDemo.java

```diff
@@ -2,19 +2,23 @@
  * 
  */
 package ConsoleApp;
-
 /**
  * @author Louis
  *
  */
+import java.util.Scanner;
+
 public class consoleAppDemo {
+	
```

Comment:
There should never be more than empty line between code. Line 12 and 13 should be condensed to 1 line.